index.html

```diff
@@ -46,5 +47,178 @@ <h1>I’m Rashmi.<br> to see<br> you!</h1>
                 </ul>
             </div>
         </div>
+
+        <section class="work">
+            <div class="grid-container">
+                <div class="grid-item">
+                    <img src="img/Snapshoot Portfolio.png">
+                    <div class="detail">
+                        <h1>Tonic</h1>
+                        <div class="list-01">
+                            <ul>
+                                <li class="canopy">CANOPY</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>Back End Dev</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
```

Comment:
According to the Figma design, there are some issues that you still need to fix to make your project as expected the design.
Please, make sure you use the right:
  - Margin & padding
  - font
  - and the background color
  
> ## The Figma design
>
><img width="431" alt="Screenshot 2023-06-03 at 6 16 03 PM" src="https://github.com/siriwardhanage/My-first-portfolio-/assets/91237525/c30def73-d73b-4a4e-807e-13f80b349c25">
>
> ## Your Design
>
><img width="373" alt="Screenshot 2023-06-03 at 6 13 28 PM" src="https://github.com/siriwardhanage/My-first-portfolio-/assets/91237525/d833e773-331e-4102-9015-9fcdd026199f">